components/treeview/stories/treeview.stories.js

```diff
@@ -0,0 +1,241 @@
+      control: "select"
+    },
+    isQuiet: {
+      name: "Quiet",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean",
+    },
+  },
+  args: {
+    rootClass: "spectrum-TreeView",
+    size: "m",
+    isQuiet: false
+  },
+  parameters: {
+    actions: {
+      handles: ["click .spectrum-TreeView-itemLink"],
+    },
+    status: {
+      type: process.env.MIGRATED_PACKAGES.includes('treeview') ? 'migrated' : undefined
+    }
+  }
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  items: [
+    {
+      id: "label1",
+      label: "Label 1",
+      link: "#",
+      isSelected: true,
+    },
+    {
+      id: "group1",
+      label: "Group 1",
+      link: "#",
+      isOpen: true,
+      items: [
+        {
+          id: "label2",
+          label: "Label 2",
+          link: "#",
+          isDisabled: true
+        },
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#"
+        },
+      ]
+    },
+    {
+      id: "group2",
+      label: "Group 2",
+      link: "#",
+      items: [
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#"
+        },
+        {
+          id: "group3",
+          label: "Group 3",
+          link: "#",
+          items: [
+            {
+              id: "label4",
+              label: "Label 4",
+              link: "#"
+            },
+          ]
+        },
+      ]
+    }
+  ]
+};
+
+export const FoldersAndFiles = Template.bind({});
+FoldersAndFiles.args = {
+  items: [
+    {
+      id: "label1",
+      label: "Label 1",
+      link: "#",
+      icon: "Document",
+    },
+    {
+      id: "group1",
+      label: "Group 1",
+      link: "#",
+      isOpen: true,
+      isSelected: true,
+      icon: "Folder",
+      items: [
+        {
+          id: "label2",
+          label: "Label 2",
+          link: "#",
+          icon: "Document",
+        },
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#",
+          icon: "Document",
+        },
+      ]
+    },
+    {
+      id: "group2",
+      label: "Group 2",
+      link: "#",
+      icon: "Folder",
+      items: [
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#",
+          icon: "Document",
+        },
+        {
+          id: "group3",
+          label: "Group 3",
+          link: "#",
+          icon: "Folder",
+          items: [
+            {
+              id: "label4",
+              label: "Label 4",
+              link: "#",
+              icon: "Document",
+            },
+          ]
+        },
+      ]
+    }
+  ]
+};
+
+export const Thumbnails = Template.bind({});
+Thumbnails.args = {
+  items: [
+    {
+      id: "group1",
+      label: "Group 1",
+      link: "#",
+      isOpen: true,
+      thumbnail: {
+        imageURL: "img/thumbnail.png",
```

Comment:
I am getting a broken image on each of these thumbnails. Are we expecting them to load locally? 
I was able to get `example-ava.png` to load but did not see why these were not.

Reply:
Probably because I didn't add them 🙃 they should be there now!